ui-tests/cypress/support/pageObjects/pages/designer/widgetconfigs/ContentWidgetConfigPage.js

```diff
@@ -0,0 +1,76 @@
+import { htmlElements } from '../../../WebElement';
+
+import Dialog from '../../../app/Dialog';
+
+import WidgetConfigPage from '../WidgetConfigPage';
+
+export class ContentListSelectModal extends Dialog {
```

Comment:
I've created an additional class to handle different dialog body contents on another [PR](https://github.com/entando/entando-test-automation/pull/14/commits/c46c59d90e5efc375884e5b129d880cabd1f4642); what do you think about it?

Reply:
looks okay. I'll implement it when that PR goes to merge and I am also working on the next task that still involves on MFE/Widgets so by that time I supposed your PR will get to merge. is that okay?

Reply:
sure! 😄